src/pytest_cookies/plugin.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, exception=None, exit_code=0, project_dir=None, context=None):
     @property
     def project(self):
```

Comment:
I think I would keep this property returning a py.path.local object for compatability and add a `path` property or similar that returns a `pathlib.Path` object.